### PR TITLE
[python] Convert non-list types to lists in has_features

### LINKED ISSUE #1185: espressomd.has_featues silently fails when passing a single feature
has_features("EXCLUSIONS") -> False
has_features(("EXCLUSIONS",)) -> True

In my opinion, the 1st syntax should be supported. If it isn't, it should throw an exception.

### PR DESCRIPTION
**Pull request by hmenke**
Fixes #1185

### CODE REVIEW

**Review by mkuron — CHANGES_REQUESTED** [review]
Better use `type(arg) is not str and hasattr(arg, "__iter__")` to check whether the argument is iterable. Only comparing against known containers (`tuple`, `list`) doesn't cover if somebody uses `set` or `numpy.ndarray(..., dtype=str)`.

**Comment by mkuron** [comment]
Also, while you're at it: could you change it to use `*args` so that the following works too: `has_features("EXCLUSIONS", "ELECTROSTATICS")`? This would remove the need for brackets.

**Comment by hmenke** [comment]
@mkuron These fixes should capture everything.

File: src/python/espressomd/__init__.py [hunk 1]
```diff
@@ -27,20 +27,26 @@
 from espressomd.system import System
 from espressomd.code_info import features
 
-def has_features(arg):
+def has_features(*args):
     """Tests whether a list of features is a subset of the compiled-in features"""
 
-    return set(arg) < set(features())
+    if len(args) == 1 and type(args[0]) is not str and hasattr(args[0], "__iter__"):
+        return set(args[0]) < set(features())\
```
Comment (mkuron) [inline src/python/espressomd/__init__.py]: unnecessary backslash at the end

**Review by mkuron — APPROVED** [review]
Looks good, except for that one unneeded backslash.

File: src/python/espressomd/__init__.py (hunk 1 above)
Comment (hmenke) [reply src/python/espressomd/__init__.py]: Well, English keyboard...

**Pull request merged**